src/dams_mcda/ui.R

```diff
@@ -564,24 +610,27 @@ ui <- shinyUI(fluidPage(
 
 			actionButton("updateBtn5", "Update")
 
-		), # end
-		#End North Twin Dam Tab
+		), # End North Twin Dam Tab
+
 
+		#----------------------------------------
+		# Dolby Dam
+		#----------------------------------------
 		tabPanel("Dam 6: Dolby",
-		  #"Dolby Dam",
-		  htmlOutput("Dam6"), # status and title
 		  h2("Dolby Dam (FERC No. P-2458)"),
-		  
+
 		  HTML(
 		    "Please consider the decision criteria listed below for Dolby Dam.<br>\
-        <br><a>https://github.com/dams-mcda/Dams-MCDA/blob/TabsUpdate_Dams/src/dams_mcda/Factsheet_PenobscotMills.pdf</a> <br>\
+
 		    <br><b>Warning: decision criteria ratings must sum to 1!</b> The tracking indicator (in the box to the right of the first decision criterion) will help you keep track of the sum. Be aware that decision criteria are directly compensating (i.e., if the sum of all ratings is 1, then\ 
 		    increasing the rating on one criterion requires another criterion rating to decrease to keep the sum equal to 1). <br>\
 		    <br> Please consider and rate the given set of decision criteria at the dam. <b>In each case, 0 = not at all important and 1 = extremely important.</b><br>"
 		  ),
-		  
+
+		  helpText("Click HERE for more information about the Penobscot Mills Project", href="https://github.com/dams-mcda/Dams-MCDA/blob/TabsUpdate_Dams/src/dams_mcda/Factsheet_PenobscotMills.pdf"),
```

Comment:
I don't like linking the pdf off the github path.

Reply:
I wasn't sure how else to do it, so I'm open to other options. We also have a folder full of maps to do this with once Sam's piece is fully integrated.